14/umbraco-cms/extending/packages/language-files-for-packages.md

```diff
@@ -2,39 +2,58 @@
 description: "Information on how to use language files to make your Umbraco package UI support multiple languages"
 ---
 
-# Language file for packages
-
-Umbraco Core includes language files, but package authors must provide their own for multi-lingual UI.
-
-## Including new language keys
-
-For each language your package supports, you include an .xml file in the same format as the core language files, named with its language code. The language files must be located in a `Lang` folder inside your package folder in `App_Plugins`. If your package assets are in `/App_Plugins/mypackage` all language files must be placed in the following locations:
-
-- English keys: `/App_Plugins/mypackage/Lang/en-US.xml`
-- Danish keys: `/App_Plugins/mypackage/Lang/da-DK.xml`
-
-{% hint style="info" %}
-The `App_Plugins` version of the `Lang` directory is case sensitive on Linux systems, so make sure that it start with a capital `L`.
-{% endhint %}
+# Languages for packages
+
+If you want your package to be available in different languages, you can use the existing localizations from Umbraco or register your own localizations. The localizations are written as a key-value pair pattern.
+To register localizations to a language, you must add a new manifest to the Extension API. The manifest can be added through the `umbraco-package.json` file like this:
+
+{% code title="umbraco-package.json" lineNumbers="true" %}
+
+```json
+{
+  ...
+  "name": "MyPackage",
+  "extensions": [
+    {
+      "type": "localization",
+      "alias": "MyPackage.Localize.EnUS",
+      "name": "English (United States)",
+      "meta": {
+        "culture": "en-us"
+      },
+      "js": "/App_Plugins/MyPackage/Localization/en-us.js"
+    }
+  ]
+}
+```
 
 ## Language file format
```

Comment:
Should this heading be removed as well? 🤔

Reply:
Yes, it should be removed :)